server/src/main/kotlin/com/mobilejazz/colloc/domain/model/Dictionary.kt

```diff
@@ -0,0 +1,7 @@
+package com.mobilejazz.colloc.domain.model
+
+typealias Dictionary = Map<Language, Translation>
+
+typealias Translation = Map<String, String>
+
+data class Language(val code: String, val name: String)
```

Comment:
I think it would be better that `Language` has its own file `Language.kt`

Reply:
The file is not too big and everything is very related. We can keep them in same class to just to avoid creating new file for them. What do you think?